skflow/__init__.py

```diff
@@ -93,8 +95,19 @@ def _setup_training(self):
             self._trainer = TensorFlowTrainer(self._model_loss,
                 self._global_step, self.optimizer, self.learning_rate)
             self._session = tf.Session(self.tf_master)
-
-    def fit(self, X, y):
+    
+    def _setup_summary_writer(self, logdir):
+        """Sets up the summary writer to prepare for later optional visualization."""
+        # Create summary to monitor loss
+        tf.scalar_summary("loss", self._model_loss)
+        # Set up a single operator to merge all the summaries
+        summary_op = tf.merge_all_summaries()
```

Comment:
I ment `summary_op`, you can just call `tf.merge_all_summaries()`. `self._summary_writer` will be useful later for reporting additional info, like predictions.